compiler/ilgen/TypeDictionary.cpp

```diff
@@ -677,10 +649,11 @@ TypeDictionary::FieldReference(const char *typeName, const char *fieldName)
    {
    TR_HashId typeID = 0;
 
-   auto found = _structsByName->locate(typeName, typeID);
+   StructIterator it = _structsByName.find(typeName);
+   bool found = it != _structsByName.end();
    if (found)
```

Comment:
You should just simplify here and eliminate the use of the `found` bool since comparing the iterator to the end "value" is idiomatic.